specification/monitor/resource-manager/Microsoft.Insights/preview/2023-11-01-preview/metricsDefinitions_API.json

```diff
@@ -0,0 +1,273 @@
+{
+    "swagger": "2.0",
+    "info": {
+      "title": "MonitorManagementClient",
+      "version": "2018-01-01"
```

Comment:
We need to decide the version for these swaggers. Can we use the old version or must we update to the new one?

Reply:
decided on the version offline with Srikanta